hass_nabucasa/auth/cognito.py

```diff
@@ -329,7 +334,7 @@ async def _async_renew_access_token(self) -> None:
             await self.cloud.run_executor(cognito.renew_access_token)
             await self.cloud.update_token(cognito.id_token, cognito.access_token)
 
-        except ClientError as err:
+        except (ClientError, EndpointConnectionError) as err:
```

Comment:
Considering `_map_aws_exception` now supports `BotoCoreError` as well, we could merge this and the next `except` into one. It'd still throw `UnknownError` for unknown Boto Core errors.

Reply:
It will change the behaviour of all that today hit unknown, as they will now start having the error passed to the constructor.